.travis.yml

```diff
@@ -24,4 +24,9 @@ before_script:
 # command to run tests
 script:
   - coverage run --source=authors manage.py test
-  - coveralls
\ No newline at end of file
+  - python manage.py test
```

Comment:
Hey cedric you reverted the changes we made for the Travis file.
can you look at the develop branch that has the latest changes to the travis file.

Reply:
Fixing it